README.md

```diff
@@ -0,0 +1,66 @@
+# Sensu Plugin Tool
+
+- [Overview](#overview)
+- [Usage examples](#usage-examples)
+- [Installation from source and
+  contributing](#installation-from-source-and-contributing)
+
+## Overview
+
+The [Sensu Plugin Tool][0] is a command-line tool to generates scaffolding for a
```

Comment:
Is it necessary to link it's own repository?

Reply:
I add it just in case the README gets parsed by any other websites (e.g. Bonsai for plugins) so there will always be a link back to the repository.

Reply:
sensu-plugin-tool is not a Sensu asset though, so it shouldn't be on Bonsai, right? Sorry if I'm missing something, I do think the intention is good, but I'm trying to visualize a real scenario that would happen in.